tests/utils.py

```diff
@@ -35,9 +35,11 @@ def ignored_warning(warning):
 
 @contextmanager
 def _tmp_file(content, extension):
-    with tempfile.NamedTemporaryFile(suffix='.' + extension, mode='w') as tmp_file:
+    with tempfile.NamedTemporaryFile(suffix='.' + extension, mode='w', delete=False) as tmp_file:
         tmp_file.write(content)
         tmp_file.seek(0)
+        tmp_file.close()
```

Comment:
Windows can not use the file if it's still open.

Reply:
the 'with' doesn't close the file?

Reply:
You're probably right. Let me try this.

Reply:
Actually, the `yield` happens within the `with` statement so the file is not closed yet.